Project SF/Views/Onboarding/GrantDataAccessView.swift

```diff
@@ -39,15 +39,15 @@ struct GrantDataAccessView: View {
                     }
                 }
             }
-            .padding()
+                .padding()
             Text("This application needs access to your health data to calculate the points in competitions.")
                 .padding(.horizontal, 16)
                 .font(.caption)
                 .foregroundColor(.secondary)
                 .multilineTextAlignment(.center)
         }
         .padding(.horizontal)
-        .navigationTitle("Health Data")
+        .navigationTitle("iCloud")
```

Comment:
Why change nav title to iCloud if you aren't doing iCloud on this page?

Reply:
Whoops, forgot to change that. I was just trying to add iCloud grant data on the onboarding.

Reply:
Fixed